LocationUpdatesBackgroundKotlin/app/src/main/java/com/google/android/gms/location/sample/locationupdatesbackgroundkotlin/data/db/MyLocationTypeConverters.kt

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (C) 2020 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.android.gms.location.sample.locationupdatesbackgroundkotlin.data.db
+
+import androidx.room.TypeConverter
+
+import java.util.Date
+import java.util.UUID
+
+/**
+ * Converts non-standard objects in the {@link MyLocation} data class into and out of the database.
+ */
+class MyLocationTypeConverters {
```

Comment:
could be named `LocationTypeConverter`

Reply:
Same here... was trying to separate from "Location" since that is a predefined Android class. Maybe I'm overthinking it too much. I could change to SimpleLocation or something.

Reply:
ok. it's a sample. let's leave it like this